docs/configuration.md

```diff
@@ -1,6 +1,6 @@
 # Configuring Ruff
 
-Ruff can be configured through a `pyproject.toml`, `ruff.toml`, or `.ruff.toml` file.
+Ruff can be configured through a `.ruff.toml`, `ruff.toml`, or `pyproject.toml` file.
```

Comment:
I can see how the current ordering can be misleading as it can give the impression that it matches the precedence in which configuration files are discovered. However, the ordering is somewhat intentional because most users should use `pyproject.toml` files over a `ruff.toml`

Reply:
Sure if there's an intended order through usage priority, let's keep it. Maybe it's reasonable to add a short paragraph on the advantages/disadvantages of the different options and when to use what. But that can be a separate topic.